db/migrate/20250614122127_improve_github_columns.rb

```
@@ -0,0 +1,19 @@
+class ImproveGithubColumns < ActiveRecord::Migration[7.1]
+  def change
+    add_index :users, :github_username, unique: true
+
+    change_column_default :reports, :github_push_status, from: nil, to: "not_pushed"
+    
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)